### ISSUE TITLE
Exception Call to a member function findAllowedByIsoCodeA3() on null in version 5.4.0

### ISSUE THREAD

**Issue by tobiasschaeferptb** [in2code-de/femanager#279]
Hi,

after an update from 5.3.1 to 5.4.0 editing of fe-users (in the backend) fails if a fe-user has already a country.
The exception is:
Core: Exception handler (WEB): Uncaught TYPO3 Exception: Call to a member function findAllowedByIsoCodeA3() on null | Error thrown in file typo3conf/ext/femanager/Classes/DataProvider/CountryZonesDataProvider.php in line 67.
static_info_tables is installed in version 6.8.0
TYPO3 is 8.7.37
PHP is 7.2.5

Cheers, Tobias

**Comment by ochorocho** [comment]
Facing the same issue. 

The countryRepository is null. So i reckon the repository is not injected properly

TYPO3 9.5.21
fenanager 5.4.0

**Comment by mediaessenz** [comment]
Same here

**Comment by mediaessenz** [comment]
Found the problem:
The Classes/UserFunc/StaticInfoTables::getStatesOptions method uses GeneralUtility::makeInstance to instanciate the CountryZonesDataProvider, which works under TYPO3 10, but not under TYPO3 9 (different kind of dependency injection).

This would do the job under TYPO3 9:
```
$objectManager = GeneralUtility::makeInstance(ObjectManager::class);
$countryZonesDataProvider = $objectManager->get(CountryZonesDataProvider::class);
```

**Comment by sbusemann** [comment]
should be resolved with 5.4.2
